src/scrabble.js

```diff
@@ -1,5 +1,98 @@
-function scrabble() {
+const scoreValues = {
+  // Corresponding alphabet to letter score values
+  A: 1,
+  E: 1,
+  I: 1,
+  O: 1,
+  U: 1,
+  L: 1,
+  N: 1,
+  R: 1,
+  S: 1,
+  T: 1,
+  D: 2,
+  G: 2,
+  B: 3,
+  C: 3,
+  M: 3,
+  P: 3,
+  F: 4,
+  H: 4,
+  V: 4,
+  W: 4,
+  Y: 4,
+  K: 5,
+  J: 8,
+  X: 8,
+  Q: 10,
+  Z: 10,
+}
 
+function isValidInput(word) {
+  return (word !== undefined && word !== null);
 }
 
-module.exports = scrabble
+function scrabble(word) {
+  if (!isValidInput(word)) {
+    return 0;
+  }
+
+  let s = 0; // stands for score 
```

Comment:
Personally I would avoid defining the variable like you have in this way where the variable is a single letter with a comment on what it means. 

It may be better to directly define the variable (score).